arch/x86/Kconfig

```diff
@@ -80,6 +80,20 @@ choice
 		  Reboot via the RST_CNT register, going back to BIOS.
 endchoice
 
+config X86_ACPI
```

Comment:
Can we just call it ACPI and make it depend on X86 or something? The X86 is either redundant (when you are adding this under arch/x86) and can't be reused if a feature appears on a different architecture, for example MULTIBOOT.

Reply:
Hmm, well the Multiboot config options are all `X86_MULTIBOOT_*`, too. I'm just trying to be consistent. I figure if the day comes that we implement ACPI on ARM (and Itanium hahah) we'll probably want to reconcile the implementations with some common interface and *then* rename the CONFIG_X86_ACPI to just CONFIG_ACPI. Does that make sense?  Right now it only applies to x86 and is pretty X86-specific.

Reply:
I would call it CONFIG_ACPI now already, no reason to namespace it like this, if it is supported only by X86, just make a dependency on that. It is cleaner this way and future proof.

Reply:
@nashif should I change X86_MULTIBOOT to MULTIBOOT as well, while I'm in here?

Reply:
renamed